plans/features/integration.fmf

```diff
@@ -0,0 +1,12 @@
+summary: Verify integration with other systems
```

Comment:
Hi, theoretically yes. I expect if there will be incorporated ReportPortal or beaker or whatever, that it could be also added into integration tests as well, so for me these integration, now It is just about nitrate, but just for now. Could be extended to any other external system integration as well.

Reply:
Thanks for moving the plan.